lib/muchos/config/base.py

```diff
@@ -73,8 +73,8 @@
   'worker_data_dirs': None,
   'zookeeper_connect': "{% for host in groups['zookeepers'] %}{{ host }}:2181{% if not loop.last %},{% endif %}{% endfor %}",
   'zookeeper_client_port': '"2181"',
-  'zookeeper_home': '"{{ install_dir }}/zookeeper-{{ zookeeper_version }}"',
-  'zookeeper_tarball': 'zookeeper-{{ zookeeper_version }}.tar.gz',
+  'zookeeper_home': "{% if zookeeper_version.startswith('3.5') %}{{ install_dir }}/apache-zookeeper-{{ zookeeper_version }}-bin{% else %}{{ install_dir }}/zookeeper-{{ zookeeper_version }}{% endif %}",
+  'zookeeper_tarball': "{% if zookeeper_version.startswith('3.5') %}apache-zookeeper-{{ zookeeper_version }}-bin.tar.gz{% else %}zookeeper-{{ zookeeper_version }}.tar.gz{% endif %}",
```

Comment:
Since the part that is different here is the same that is different in the `zookeeper_home` variable, it might make sense to put that in a new variable that can be injected into these two.

Not sure if this would work, but maybe something like:

```suggestion
  'zk_naming': "{% if zookeeper_version.startswith('3.5') %}apache-zookeeper-{{ zookeeper_version }}-bin{% else %}zookeeper-{{ zookeeper_version }}{% endif %}",
  'zookeeper_home': "{{ install_dir }}/{{ zk_naming }}",
  'zookeeper_tarball': "{{ zk_naming }}.tar.gz",
```